src/zope/testrunner/formatter.py

```diff
@@ -1334,3 +1346,293 @@ def _emit_exists(self, test):
         self._subunit.status(
             test_id=test.id(), test_status='exists',
             test_tags=self._subunit.current_tags, timestamp=now)
+
+
+class TestSuiteInfo(object):
+
+    def __init__(self):
+        self.testCases = []
+        self.errors = 0
+        self.failures = 0
+        self.time = 0.0
+
+    @property
+    def tests(self):
+        return len(self.testCases)
+
+    @property
+    def successes(self):
+        return self.tests - (self.errors + self.failures)
```

Comment:
```suggestion
        return self.tests - self.errors - self.failures
```

I think this is easier to grasp.